src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,44 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+
+  constructor(hidden, health) {
+    super(hidden, health);
+    this.hidden = false;
+    this.health = 100;
+  }
+
```

Comment:
Don't need this constructor, because this class already inherits the constructor from Animal.
write only : 
hidden = false;
health move to Animal constructor